code/game/objects/items/storage/backpack.dm

```diff
@@ -631,6 +631,7 @@
 	new /obj/item/clothing/suit/armor/vest/rurmcoat(src)
 	new /obj/item/clothing/suit/armor/vest/sovietcoat(src)
 	new /obj/item/clothing/under/yogs/hopcasual(src)
+	new /obj/item/clothing/shoes/xeno_wraps/command(src)
```

Comment:
Remove this, HoP is blacklisted from xenos - only the four lower Heads should have clothing bag additions